_rules/link-skip-block-content-7b576d.md

```diff
@@ -0,0 +1,52 @@
+---
+id: 7b576d
+name: Link for skipping block of content
+rule_type: atomic
+description: |
+  This rule checks that blocks of content can be skipped by a link at their beginning
+accessibility_requirements:
+  wcag-technique:G123: # Adding a link at the beginning of a block of repeated content to go to the end of the block
+		forConformance: false
+		failed: not satisfied
+		passed: further testing needed
+		inapplicable: further testing needed
+input_aspects:
+	- DOM Tree
+	- CSS styling
+	- Language
+authors:
+	- Jean-Yves Moyen
+	- Anne Thyme Nørregard
+---
+
+## Applicability
+
+This rule applies to any [document](#https://www.w3.org/TR/dom/#concept-document) where the [document element](#https://www.w3.org/TR/dom/#document-element) is an HTML `html` element.
+
+## Expectations
+
+For each [section of repeated content](#repeated-content) within the test target, either the last [focusable](#focusable) element before it, or the first [focusable](#focusable) element inside it has a [semantic role](#semantic-role) of `link` and:

```

Comment:
I think you need to better specify what is the order when you're stating "the last focusable element before it" or "the first focusable element inside it". Is it the last focusable element before the first focusable element that is inside the section of content (I assume so) or the last focusable element in visual order before the section (e.g. you have a focusable element that lies just above the section, but your next focusable element goes below the section, and only the one after that is inside the section?